### PR TITLE
Documentation Website

### PR DESCRIPTION
**Pull request by fkorotkov**
Our ReadMe is too big now. It's time for a separate website with documentation:

https://user-images.githubusercontent.com/989066/212981141-57dbfa5d-3cfc-460f-ae3f-7bdfc26d797d.mov

### CODE REVIEW

**Comment by fkorotkov** [comment]
Feel free to try things around locally via:

```bash
docker run --pull --rm -it -p 8000:8000 -v ${PWD}:/docs ghcr.io/squidfunk/mkdocs-material:latest
```

File: docs/theme/overrides/home.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        <p>
+          Tart utilizes <i>Virtualization.Framework</i> to achieve nearly native performance.
+          Benefit from clean ephemeral environments for your automations with no performance implications.
+        </p>
+        <p>
+          For storing virtual machine images Tart integrates with OCI-compatible container registries.
+          Push and pull virtual machines like you used to with Docker containers.
+        </p>
+        <p>
+          Tart powers several continuous integration systems including <a href="/cirrus-runners">on-demand GitHub Actions</a>
```
Comment (edigaryev) [inline docs/theme/overrides/home.html:67]:
```suggestion
          Tart powers several continuous integration systems including <a href="/github-actions">on-demand GitHub Actions</a>
```

**Comment by edigaryev** [comment]
LGTM, however, there's now an issue with maintaining the `README.md` and the website (no DRY).

Are we going to remove something from the README.md, or we'll try to maintain both variants anyway?

**Comment by fkorotkov** [comment]
I moved most of the things from ReadMe. The only duplicate is `Usage` part which is 4 lines. IMO we should keep the ReadMe small and try to not add any more things. Website should be the source of documentation since we have more flexibility there. Made the ReadMe even smaller in [3763f00](https://github.com/cirruslabs/tart/pull/380/commits/3763f00fb49ab47299236935316e67feeb294e7b)

**Review by edigaryev — APPROVED** [review]

**Pull request merged**